lib/generators/hyrax/work_resource/templates/indexer.rb.erb

```diff
@@ -3,7 +3,7 @@
 # Generated via
 #  `rails generate hyrax:work_resource <%= class_name %>`
 class <%= class_name %>Indexer < Hyrax::ValkyrieWorkIndexer
-  Hyrax::ValkyrieIndexer.register self, as_indexer_for: <%= class_name %>
+  include Hyrax::Indexer(:basic_metadata)
```

Comment:
With this change, will those that ran previous generators need to change their registration logic? What is the process for people to do that? Do we have documentation in the upgrade notes?

Reply:
> With this change, will those that ran previous generators need to change their registration logic?

yes.

> What is the process for people to do that? Do we have documentation in the upgrade notes?

this code has never been released (even in an RC), so i think we might be in the clear here? i'm open to counter views on that point, though.

Reply:
i guess i'll also add: they only need to make this change if they want `:basic_metadata`.

if they *do* want that, it's likely they've already made this change.